tests/integration_tests/functional/test_cmd_line_start.py

```diff
@@ -104,3 +104,41 @@ def test_config_start_no_api(test_microvm_with_api, vm_config_file):
         exceptions=RuntimeError,
         tries=10,
         delay=1)
+
+
+@pytest.mark.parametrize(
+    "vm_config_file",
+    ["framework/vm_config.json"]
+)
+def test_config_start_with_limit(test_microvm_with_api, vm_config_file):
+    """
+    Test if a microvm configured from file boots successfully.
+
+    @type: functional
+    """
+    test_microvm = test_microvm_with_api
+
+    _configure_vm_from_json(test_microvm, vm_config_file)
+    test_microvm.jailer.extra_args.update({'payload_size': "250"})
```

Comment:
I would also like us to test the default limit value. Let's test a request with payload within bounds and one with content length greater than the default limit.